openapi/data_repository_service.swagger.yaml

```diff
@@ -203,8 +203,8 @@ definitions:
       self_uri:
         type: string
         description: |-
-          A drs:// URI, as defined in the DRS documentation, that tells clients how to access this object.
-          The intent of this field is to make DRS objects self-contained, and therefore easier for clients to store and pass around.
+          A drs:// hostname-based URI, as defined in the DRS documentation, that tells clients how to access this object.
+          The intent of this field is to make DRS objects self-contained, and therefore easier for clients to store and pass around.  For example, if you arrive at this DRS JSON by resolving a compact identifier-based DRS URI, the `self_uri` presents you with a hostname and properly encoded DRS ID for use in subsequent `access` endpoint calls.
```

Comment:
If I'm reading this correctly, the self_uri is always a hostname-based uri?

Reply:
Right now yes. I opened a new issue to explore the possibility of including a self_url in the response since that would make it even easier to construct the "access" request.